packages/terra-core-docs/CHANGELOG.md

```diff
@@ -5,16 +5,19 @@
 * Removed
   * Removed custom styles of `terra-arrange` examples to make whole content visible.
 
-## 1.37.0 - (August 18, 2023)
-
 * Added
+  * Added drag and drop example for `terra-list`.
   * Added accessibility guidance for `terra-arrange`.
   * Added an event to prevent page scroll on hyperlink for `terra-toggle-section-header`.
+
+## 1.37.0 - (August 18, 2023)
+
+* Added
```

Comment:
Please rebase with main so this is no longer part of your PR.

Reply:
Rebased the branch with main. Thanks